README.md

```diff
@@ -17,104 +17,139 @@ This documentation outlines the necessary and optional [dependencies](#dependenc
   Downloaded automatically if absent.
+- [Boost](https://www.boost.org/) v1.78.0: A set of free peer-reviewed
+  portable C++ libraries. Downloaded automatically by conan.
+- [fftw3](www.fftw.org) v3.3.9: Fastest Fourier Transform in the West. Downloaded automatically by conan.
+- [Eigen3](http://eigen.tuxfamily.org/index.php?title=Main_Page) v3.3.7: Modern `C++` linear algebra. Downloaded automatically by conan.
+- [tiff](http://www.libtiff.org/) v4.0.9: Tag Image File Format library. Downloaded automatically by conan.
+- [cfitsio](http://heasarc.gsfc.nasa.gov/fitsio/fitsio.html): v4.0.0: Library of `C` and `Fortran` subroutines for reading and writing data files in FITS (Flexible Image Transport System) data format. Downloaded automatically by conan.
+- [yaml-cpp](https://github.com/jbeder/yaml-cpp) v0.6.3: YAML parser and emitter in `C++`. Downloaded automatically by conan.
 - [casacore](http://casacore.github.io/casacore/) - Optional - Needed to interface with measurement
-  sets. The main purify program requires this library (and its dependencies)
-- [OpenMP](http://openmp.org/wp/) v4.8.4 (Trusty) - Optional - Speeds up some of the operations.
-- [MPI](https://www.open-mpi.org) v3.1.1 (Trusty) - Optional - Parallelisation paradigm to speed up operations.
-- [spdlog](https://github.com/gabime/spdlog) v* - Optional - Logging library. Downloaded automatically if
-  absent.
-- [Catch2](https://github.com/catchorg/Catch2) v2.3.0 - Optional -  A `C++`
-  unit-testing framework only needed for testing. Downloaded automatically if absent.
-- [google/benchmark](https://github.com/google/benchmark) `v1.3.0` - Optional - A `C++`
-  micro-benchmarking framework only needed for benchmarks. Downloaded automatically if absent.
+  sets. The main **PURIFY** program requires this library (and its dependencies)
+- [spdlog](https://github.com/gabime/spdlog) v1.9.2: Optional - Logging library. Downloaded automatically by conan.
+- [Catch2](https://github.com/catchorg/Catch2) v2.13.9: Optional -  A `C++`
+  unit-testing framework only needed for testing. Downloaded automatically by conan.
+- [google/benchmark](https://github.com/google/benchmark) v1.6.0: Optional - A `C++`
+  micro-benchmarking framework only needed for benchmarks. Downloaded automatically by conan.
 
 Installing and building PURIFY
 -------------------------------------
 
-**PURIFY** can be installed through the software packet manager on Linux Debian distributions:
+To build **PURIFY**:
 
-```
-apt-get install purify
-```
+1. Once the mandatory user-provided dependencies are present, `git clone` from the [GitHub repository](https://github.com/astro-informatics/purify):
 
-Alternatively, you can build **PURIFY** entirely from the source code. Once the mandatory dependencies are present, `git clone` from the [GitHub repository](https://github.com/astro-informatics/purify):
+    ``` bash
+    git clone --recurse-submodules https://github.com/astro-informatics/purify.git
```

Comment:
This step created the `sopt` folder for me but didn't actually fetch the files, I had to add a `git submodule update --init --recursive` step. Is this the same for you @tkoskela @20DM ?

Reply:
I think this is because it does not recuse submodules in all branches, only the one it pulls when you clone. If I do
```
git clone --recurse-submodules --branch sj/conan-build https://github.com/astro-informatics/purify.git
```
I get all the submodules included, so I assume once this is merged into the default branch the current documentation will work.